openlibrary/plugins/upstream/merge_authors.py

```diff
@@ -85,30 +129,27 @@ def merge_property(self, a, b):
         else:
             return a
 
-    def make_redirect_doc(self, key, redirect):
-        return {
-            "key": key,
-            "type": {"key": "/type/redirect"},
-            "location": redirect
+
+class AuthorRedirectEngine(BasicRedirectEngine):
+    def find_backreferences(self, key):
```

Comment:
To clarify my confusion and (hopefully!) show I'm not just being pedantic, but I though a method named `find_backreferences()` or `get_backreferences()` would return a list of other types and fields. Those types could then be searched in the associated fields for a particular key.  That's what the current data model uses that name for anyway.

Reply:
Fixed